### ISSUE TITLE
composite-metazoan and AEO

### ISSUE THREAD

**Issue by cmungall** [obophenotype/uberon#1957]
CM includes:

```
id: AEO:0001000
name: sensory ganglion
def: "A ganglion containing sensory neurons." [JB:AEO]
is_a: AEO:0000135 ! ganglion
is_a: UBERON:0003714 ! neural tissue
relationship: has_part CL:0000101 ! sensory neuron
relationship: has_part CL:0002573 ! Schwann cell
```

First, there is no use case for including AEO explicitly, as it is redundant with uberon/caro. However, it comes in by way of EHDAA2

if we do bring it in, we need to make sure it gets merged

we have the xref

```
id: UBERON:0001800
name: sensory ganglion
def: "The clusters of neurons in the somatic peripheral nervous system which contain the cell bodies of sensory nerve axons, interneurons and non-neuronal supporting cells." [MP:0000960]
subset: pheno_slim
synonym: "ganglion sensorium" EXACT LATIN []
xref: AEO:0001000
xref: BTO:0005630
xref: EMAPA:36596
xref: FMA:5885
xref: MA:0002566
xref: MESH:D017950
xref: NCIT:C13060
xref: UMLS:C0206429 {source="ncithesaurus:Sensory_Ganglion"}
intersection_of: UBERON:0000045 ! ganglion
intersection_of: extends_fibers_into UBERON:0001027 ! sensory nerve
```

and the header

```
treat-xrefs-as-equivalent: AEO
```

however, I believe the bridge axioms are not brought in during CM building

**Comment by cmungall** [comment]
I think we need to fix this

**Comment by gouttegd** [comment]
> however, I believe the bridge axioms are not brought in during CM building

Yes, `uberon-bridge-to-aeo.owl` is currently _not_ used anywhere when building composite-metazoan. It is not imported in any of the `collected-*` ontologies.

I’ll fix that during the overhaul of the CM pipeline.

**Comment by gouttegd** [comment]
Reopening because the use of AEO by EHDAA2 is causing some issues with `composite-metazoan`.

Consider the following EHDAA2 term:

```
[Term]
id: EHDAA2:0004406
name: pons choroid plexus capillaries
is_a: AEO:0000209 ! vein
```

AEO:0000209 is mapped to UBERON:0001638 (“vein”). EHDAA2:0004406 itself is not mapped to anything in UBERON, and is not a subclass of anything else than AEO:0000209.

Because we treat AEO terms as strictly equivalent to their mapped UBERON terms (`treat-xrefs-as-equivalent: AEO`), there is _nothing_ in the merged composite-metazoan ontology to indicate that `EHDAA2:0004406` is a human-specific term (apart from its ID of course, but that requires external knowledge that EHDAA2 terms are human-specific – it’s not something that is “encoded” in the ontology).

Of all the taxon-specific ontologies that make up composite-metazoan, EHDAA2 seems to be the only one to use AEO, so in effect AEO terms are only ever used, at least within composite-metazoan, to refer to human structures. So I would like to change the Uberon-AEO bridge to make it a “human-specific bridge“, as for the EHDAA2 bridge. This would make, in this case for example, make AEO:0000209 equivalent to `UBERON:0001638 and (part_of NCBITaxon:9606)`, and therefore ensure that EHDAA2:0004406 is properly linked to its taxon.
